File: app/src/main/java/com/example/yelpbusinessexample/data/model/Location.java
package com.example.yelpbusinessexample.data.model;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class Location{

	@SerializedName("country")
	private String country;

	@SerializedName("address3")
	private String address3;

	@SerializedName("address2")
	private String address2;

	@SerializedName("city")
	private String city;

	@SerializedName("address1")
	private String address1;

	@SerializedName("display_address")
	private List<String> displayAddress;

	@SerializedName("state")
	private String state;

	@SerializedName("zip_code")
	private String zipCode;

	public void setCountry(String country){
		this.country = country;
	}

	public String getCountry(){
		return country;
	}

	public void setAddress3(String address3){
		this.address3 = address3;
	}

	public String getAddress3(){
		return address3;
	}

	public void setAddress2(String address2){
		this.address2 = address2;
	}

	public String getAddress2(){
		return address2;
	}

	public void setCity(String city){
		this.city = city;
	}

	public String getCity(){
		return city;
	}

	public void setAddress1(String address1){
		this.address1 = address1;
	}

	public String getAddress1(){
		return address1;
	}

	public void setDisplayAddress(List<String> displayAddress){
		this.displayAddress = displayAddress;
	}

	public List<String> getDisplayAddress(){
		return displayAddress;
	}

	public void setState(String state){
		this.state = state;
	}

	public String getState(){
		return state;
	}

	public void setZipCode(String zipCode){
		this.zipCode = zipCode;
	}

	public String getZipCode(){
		return zipCode;
	}
}

File: app/src/main/java/com/example/yelpbusinessexample/data/model/Location.kt
package com.example.yelpbusinessexample.data.model

import com.google.gson.annotations.SerializedName

data class Location(
    @SerializedName("country") var country: String? = null,

    @SerializedName("address3") var address3: String? = null,

    @SerializedName("address2") var address2: String? = null,

    @SerializedName("city") var city: String? = null,

    @SerializedName("address1") var address1: String? = null,

    @SerializedName("display_address") var displayAddress: List<String>? = null,

    @SerializedName("state") var state: String? = null,

    @SerializedName("zip_code") var zipCode: String? = null
)